packages/client/src/recommendations/getProductRecommendations.ts

```diff
@@ -0,0 +1,28 @@
+import client, { adaptError } from '../helpers/client';
+import join from 'proper-url-join';
+import type { GetProductRecommendations } from './types';
+
+/**
+ * Method responsible for retrieving data from recommendations endpoint on
+ * MKT API.
+ *
+ * @param query - Query object with product recommendation parameters to apply.
+ * @param config - Custom configurations to send to the client
+ * instance (axios).
+ *
+ * @returns Promise object.
+ */
+
+const getProductRecommendations: GetProductRecommendations = (
+  query,
```

Comment:
This is a breaking change, however the commit message does not contain the BREAKING MESSAGE description containing the necessary changes to migrate to this new version. Also do not forget to update the [migration guide in Confluence](https://gitlab.fftech.info/fps/platform/box/wikis/@fps-core/Migration-guide-from-v1.0.0-to-v2.0.0) with the same information.

Reply:
Added Breaking change label to pull request.